### DISCUSSION TITLE
TV5 UNIS MONTREAL

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#749] [Q&A]
### Channel Name

TV5 UNI MONTREAL

### Channel ID

TV5QuebecCanada.ca

### Country

CANADA

### Language

FRENCH

### Website

https://www.tv5unis.ca/

**Comment by Sphinxroot** [comment]
cable channel

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19727
